deps/ledger-zxlib/app/common/view_s.c

```diff
@@ -131,76 +155,139 @@ const bagl_element_t *view_prepro(const bagl_element_t *element) {
     return element;
 }
 
-void h_review_button_left() {
-    h_paging_decrease();
+const bagl_element_t *view_prepro_idle(const bagl_element_t *element) {
+    switch (element->component.userid) {
+        case UIID_ICONLEFT:
+        case UIID_ICONRIGHT:
+            return NULL;
+    }
+    return element;
+}
 
+void h_review_update() {
     zxerr_t err = h_review_update_data();
     switch(err) {
         case zxerr_ok:
             UX_DISPLAY(view_review, view_prepro);
             break;
-        case zxerr_no_data:
```

Comment:
Is this outcome no longer possible?

Reply:
Again zxlib internals. They seem to remove the possibility to review decision in case of error and just show it. Seems like we do not use this feature at all (grep did not found any use of this function).